pkg/plugin/healthz.go

```diff
@@ -84,6 +98,38 @@ func (h *HealthZ) ServeHTTP(w http.ResponseWriter, _ *http.Request) {
 		http.Error(w, "plain text mismatch after decryption", http.StatusInternalServerError)
 		return
 	}
+
+	// v2 checks
+	uid := string(uuid.NewUUID())
```

Comment:
So we can tell that this UUID was generated by us and not the API server.

```suggestion
	uid := "local-healthz-check-" + string(uuid.NewUUID())
```